requirements.txt

```diff
@@ -1,2 +1,3 @@
 click
-synapseclient==2.0.0
+synapseclient==2.3.0
+pyyaml
```

Comment:
I just tried to set up a local installation of `synapseformation` using `python setup.py develop`, but `pyyaml` doesn't get installed, presumably because it's listed under `install_requires` in `setup.py`. How do you typically ensure that `install_requires` stays in sync with `requirements.txt`?

Reply:
Ah interesting, I do `pip install -e .` which reads from the `requirements.txt`